src/ipfs.io/docs/install/index.html

```diff
@@ -10,118 +10,91 @@
-
-Then, unzip the archive, and move `ipfs` somewhere in your `$PATH`:
-
-```
-mv ipfs /usr/local/bin/ipfs
+```sh
+$ tar xvfz go-ipfs_v0.3.11_darwin-amd64.tar.gz
+$ mv go-ipfs/ipfs /usr/local/bin/ipfs
 ```
 
 Test it out:
 
-```
-> ipfs version
-ipfs version 0.3.8
-```
-
-That's it!
-
-<a class="btn btn-success btn-lg" href="../getting-started" role="button">
-  Getting Started with IPFS &nbsp;&nbsp;<i class="fa fa-arrow-right"></i>
-</a>
-
-
-Note: if you're concerned about security, build from source, as either
-[gobuilder.me](https://gobuilder.me) or [github.com](https://github.com)
-could introduce vulnerabilites. We will be doing signed releases soon.
-[Post here](https://github.com/ipfs/go-ipfs/issues/957) if you'd like this
-sooner.
-
-<br />
-
-## From Source
-
-Dependencies:
-
-- [Install Go 1.5](https://golang.org/doc/install) (to compile. [make sure it's 1.5](#check-go-version))
-- [Install Git](https://git-scm.com/) (to download dependencies)
-- *Optionally*, install [FUSE](#install-fuse) (for mounting / advanced use)
-
-Then run:
-
 ```sh
-go get -u github.com/ipfs/go-ipfs/cmd/ipfs
+$ ipfs version
+  ipfs version 0.3.11
 ```
 
-This will download ipfs and its dependencies, and compile `ipfs`.
+Congratulations! You now have a working IPFS installation on your computer.
 
-> Note: If you're new to Go, you'll need to do a few things to make sure you setup your `PATH` and your `GOPATH` (the location Go looks for libraries):
-> - Create a folder. e.g. `mkdir ~/go`
-> - Add the following to your shell config file (`.bashrc`, `.profile`, etc.)
->   - `export GOPATH=$HOME/go;`
->   - `export PATH="$GOPATH/bin:$PATH:";`
->   - `export PATH="$PATH:/usr/local/opt/go/libexec/bin";`
+### Windows
+
+After downloading, unzip the archive, and move `ipfs.exe`  somewhere in your `%PATH`.
 
 Test it out:
 
-```
+```sh
 > ipfs version
-ipfs version 0.3.8
+  ipfs version 0.3.11
 ```
 
-That's it!
+Congratulations! You now have a working IPFS installation on your computer.
+
+---
 
-<a class="btn btn-success btn-lg" href="../getting-started" role="button">
-  Getting Started with IPFS &nbsp;&nbsp;<i class="fa fa-arrow-right"></i>
-</a>
+## Building from Source
 
-<br />
+If you want, you can also build IPFS from source.
+If you are on Mac OS X or Linux take a look at the [readme](https://github.com/ipfs/go-ipfs#build-from-source) for install instructions.
+If you are on Windows take a look at this [document](https://github.com/ipfs/go-ipfs/blob/master/docs/windows.md) for instructions.
 
 ---
 
-### Troubleshooting
+## Troubleshooting
 
-#### Help!
+### Help!
 
 If you have any problems, come get live help at
 [#ipfs](../#community) or via [the mailing list](../#community).
 
-#### Check Go Version
+### Check Go Version
 
-IPFS works with Go 1.5+.
+IPFS works with Go 1.5.2+.
 To check what go version you have installed, type `go version`.
 Here's what I get:
 
 ```sh
-> go version
-go version go1.5.1 darwin/amd64
+$ go version
```

Comment:
please don't change things to `$`. You may use `$`, but `>` is shell-neutral. (Also, more interpreters use `>` across all computing environments than `$`.)